scripts/amex_browseros_bridge.py

```
@@ -0,0 +1,543 @@
+    if hostname is None or not (
+        hostname == AMEX_HOST_SUFFIX or hostname.endswith(f".{AMEX_HOST_SUFFIX}")
+    ):
+        raise RuntimeError("Refusing to fill credentials outside americanexpress.com")
+    return url
+
+
+def is_activity_page(snapshot: str) -> bool:
+    return urlparse(amex_url(snapshot)).path.rstrip("/") == "/activity"
+
+
+def download_path(response: Any) -> Path:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `response`

(ANN401)